lib/shared/addon/components/cluster-driver/driver-oracleoke/component.js

```diff
@@ -0,0 +1,448 @@
+import Component from '@ember/component'
+import ClusterDriver from 'shared/mixins/cluster-driver';
+import layout from './template';
+import { equal } from '@ember/object/computed'
+
+import {
+  get, set, computed, observer, setProperties
+} from '@ember/object';
+import { inject as service } from '@ember/service';
+
+const regionMap = {
+  'Mumbai':    'ap-mumbai-1',
+  'Seoul':     'ap-seoul-1',
+  'Tokyo':     'ap-tokyo-1',
+  'Toronto':   'ca-toronto-1',
+  'Frankfurt': 'eu-frankfurt-1',
+  'Zurich':    'eu-zurich-1',
+  'Sao Paolo': 'sa-saopaulo-1',
+  'London':    'uk-london-1',
+  'Ashburn':   'us-ashburn-1',
+  'Phoenix':   'us-phoenix-1',
+}
+
+const k8sVersionMap = {
+  'v1.15.7': 'v1.15.7', // default
+  'v1.14.8': 'v1.14.8',
+}
+
+const vcnIdMap = { quick: 'Quick Create', }
+
+const subnetAccessMap = {
+  public:  'Public',
+  private: 'Private',
+}
+
+const nodeShapeMap = {
+  'VM.Standard1.1':          'VM.Standard1.1',
+  'VM.Standard1.2':          'VM.Standard1.2',
+  'VM.Standard1.4':          'VM.Standard1.4',
+  'VM.Standard1.8':          'VM.Standard1.8',
+  'VM.Standard1.16':         'VM.Standard1.16',
+  'VM.Standard2.1':          'VM.Standard2.1',
+  'VM.Standard2.2':          'VM.Standard2.2',
+  'VM.Standard2.4':          'VM.Standard2.4',
+  'VM.Standard2.8':          'VM.Standard2.8',
+  'VM.Standard2.16':         'VM.Standard2.16',
+  'VM.Standard2.24':         'VM.Standard2.24',
+  'BM.Standard.E2.64':       'BM.Standard.E2.64',
+  'BM.Standard2.52':         'BM.Standard2.52',
+  'BM.Standard.B1.44':       'BM.Standard.B1.44',
+  'BM.DenseIO2.52':          'BM.DenseIO2.52',
+  'BM.HPC2.36':              'BM.HPC2.36',
+  'VM.Standard.E2.1.Micro':  'VM.Standard.E2.1.Micro',
+  'VM.Standard.E2.2':        'VM.Standard.E2.2',
+  'VM.GPU2.1':               'VM.GPU2.1',
+  'VM.GPU2.2':               'VM.GPU2.2',
+  'VM.GPU3.1':               'VM.GPU3.1',
+  'VM.GPU3.2':               'VM.GPU3.2',
+  'VM.GPU3.4':               'VM.GPU3.4',
+  'VM.GPU3.8':               'VM.GPU3.8',
+}
+
+const imageMap = {
+  'Oracle-Linux-7.6': 'Oracle-Linux-7.6',
+  'Oracle-Linux-7.5': 'Oracle-Linux-7.5',
+  'Oracle-Linux-7.4': 'Oracle-Linux-7.4',
+}
+
+const languages = {
```

Comment:
This can just go in the regular `translations/ en-us` file now instead of dynamically loading in here.